charts/redpanda/chart_test.go

```diff
@@ -139,6 +139,17 @@ func TestTemplate(t *testing.T) {
 			})
 			require.NoError(t, err)
 
+			// kube-lint template file
+			err = os.WriteFile("temp-template.yaml", out, 0655)
+			require.NoError(t, err)
+
+			out, errKubeLinter := exec.CommandContext(ctx, "kube-linter", "lint", "temp-template.yaml").CombinedOutput()
+			if errKubeLinter != nil {
+				t.Logf("kube-linter error(s) found: \n%s", string(out))
+			}
```

Comment:
```suggestion
                         var stdout bytes.Buffer
	                 var stderr bytes.Buffer
                         inputYaml := bytes.NewBuffer(out)
                         
			cmd := exec.CommandContext(ctx, "kube-linter", "lint", "-", "--format", "json")
			cmd.Stdin = inputYaml
	                 cmd.Stdout = &stdout
	                 cmd.Stderr = &stderr
			
			stdout, stderr, errKubeLinter := cmd.Run()
			if errKubeLinter != nil {
				t.Logf("kube-linter error(s) found: \n%s\nstderr:\n%s", stdout.String(), stderr.String())
			}
```

Thanks to that change there will not be dandling `temp-template.yaml` file and stdout can be marshalled to json for later process.

Reply:
We could also pass in the path of the golden file to `kube-linter`. Stdin works just as well though!

Reply:
cmd.Run has only one output? where are you getting 3?